src/context/SyntheticsEvents/utils.ts

```diff
@@ -15,6 +15,19 @@ export function getPendingOrderKey(data: Omit<PendingOrderData, "txnType">) {
 }
 
 export function getPendingDepositKey(data: PendingDepositData) {
+  if (data.isGlvDeposit) {
+    return [
+      data.account,
```

Comment:
we need to also include glv address as well here, different glvs can potentially share same markets